dapps/assets/src/components/contacts/Contacts.ts

```diff
@@ -175,4 +175,9 @@ export default class ContactsComponent extends mixins(EvanComponent) {
     this.filterBy = [];
     this.filter = null;
   }
+
+  filterBySearchTerm(searchTerm: string): void {
+    this.filterBy = [];
```

Comment:
would it also work just to keep the old filters and filter e.g. combined over favorites and search term?

Reply:
That's not how the filter works. It would filter inside the filterType (e.g. favorites [true, false, true])